src/server/main_service.cc

```diff
@@ -749,28 +745,32 @@ OpStatus CheckKeysDeclared(const ConnectionState::ScriptInfo& eval_info, const C
   return OpStatus::OK;
 }
 
-bool Service::VerifyCommand(const CommandId* cid, CmdArgList args, ConnectionContext* dfly_cntx) {
-  ServerState& etl = *ServerState::tlocal();
-
+optional<facade::ErrorReply> Service::VerifyCommand(const CommandId* cid, CmdArgList args) {
   string_view cmd_str = ArgS(args, 0);
 
-  absl::Cleanup multi_error([exec_info = &dfly_cntx->conn_state.exec_info] {
-    if (exec_info->IsCollecting()) {
-      exec_info->state = ConnectionState::ExecInfo::EXEC_ERROR;
-    }
-  });
-
   if (cid == nullptr) {
-    (*dfly_cntx)->SendError(StrCat("unknown command `", cmd_str, "`"), "unknown_cmd");
-
     lock_guard lk(mu_);
     if (unknown_cmds_.size() < 1024)
       unknown_cmds_[cmd_str]++;
-    return false;
+
+    return facade::ErrorReply{StrCat("unknown command `", cmd_str, "`"), "unknown_cmd"};
```

Comment:
can you add `using facade::ErrorReply` at the beginning? 
too much horizontal space is wasted on these namespaces.